app/presenters/firm_profile_presenter.rb

```diff
@@ -0,0 +1,7 @@
+class FirmProfilePresenter < SimpleDelegator
+  def type_of_advice_method
+    return :face_to_face if in_person_advice_methods.present?
+
+    :remote
+  end
+end
```

Comment:
Is there any case on our data that the firm in_person_advice_methods or other_advice_methods are empty?

Reply:
Having investigated. There are some firms that have no `in_person_advice_methods` or `other_advice_methods` - all of these (except one) are not registered and so do not appear in the search.

The only one that has no advice methods and DOES appear in the search is this one. 
https://directory.moneyadviceservice.org.uk/en/firms/690
This ends up showing the 'phone or online dialogue' - I don't know what the functionality should be for this